packages/components/pie-modal/package.json

```diff
@@ -36,7 +36,7 @@
   "license": "Apache-2.0",
   "devDependencies": {
     "@custom-elements-manifest/analyzer": "0.9.0",
-    "@justeat/pie-design-tokens": "6.3.1",
+    "@justeat/pie-design-tokens": "6.5.0",
```

Comment:
same here

Reply:
For pie-modal it is needed for styling overlay, see [here](https://github.com/justeattakeaway/pie/blob/main/packages/components/pie-modal/src/modal.scss#L123)

Reply:
I removed the usage and removed the dependency